### ISSUE TITLE
Setting up anaconda environment with QuantEcon.py

### ISSUE THREAD

**Issue by duncanhobbs** [QuantEcon/QuantEcon.py#510]
In the [instructions](https://quantecon.org/wiki-py-conda-dev-env) to set up quantecon, you may want to specify that the version of python in the environment needs to be at least python 3.5.

When I initially tried to set up a conda environment for quantecon, I ran into some trouble

When I tried to run `conda create -n quantecon-dev anaconda` for some reason it tried to install python 2.7 versions of all of the dependencies, even though I have python 3.7.2 on my machine.

When I ran `conda create -n quantecon-dev python=3.7` everything worked fine.

I am running a 32-bit version of Windows 7. (I realize most people are not running something this old!)

It might make sense to include a `python_requires=">= 3.5"` statement in the `setup()`  function in  `setup.py` [here](https://github.com/QuantEcon/QuantEcon.py/blob/master/setup.py#L96).

It might also work to test the version of python in the setup.py file before the `setup()`  function with something like this:

```
import sys
if not sys.version_info[0] == 3 and sys.version_info[1] >= 5:
    sys.exit("Python 3.5 or higher is required.")
```

**Comment by jstac** [comment]
Thanks for the report and suggestion @duncanhobbs .  When you say that you have Python 3.7 on your machine, does that mean you have a Python 3.7 version of Anaconda, such as the [current version](https://www.anaconda.com/distribution/)?  

(If so, I'm surprised that it would install Python 2.7 dependencies.  If not, then perhaps we just need to add a note in the instructions stating that the latest version of Anaconda is assumed.)

CC @mmcky

**Comment by duncanhobbs** [comment]
When I ran `conda list anaconda$` in terminal I see that I am running `version 2018.12` and build py37_0.

So its not the latest version, but I am running a 3.7 build. 

When I ran `python --version` it says that I only have python 3.7.2 installed.

I realized I am actually running 64 bit windows machine. I got confused when python uses `win32` but I realize that is just python's way of saying it runs desktop Windows. I don't think this affects anything.

So I am not sure what it was doing, but explicitly telling it to use python 3.7 when setting up the conda environment made it install the correct versions of the dependencies rather than the 2.7 versions.

@jstac @mmcky

**Comment by mmcky** [comment]
thanks @duncanhobbs I can't replicate this issue on `mac` or `linux`. When using

```
conda create -n quantecon-dev anaconda
```

it should select the python version of your base installation of `anaconda`. Perhaps that is because my base is `2019.07`. Unless you have specific version requirements for packages you may want to upgrade `conda` and `anaconda` and try again?

**Comment by duncanhobbs** [comment]
I ran `conda update conda` and then `conda update anaconda` in the anaconda prompt.

I am now running conda version 4.7.12 and a `custom` version of anaconda on the `py37_1` build.

When I tried to set up a quantecon environment using [the instructions](https://quantecon.org/wiki-py-conda-dev-env), the installer is still trying to install python 2.7 versions of all the dependencies.

I might try uninstalling and reinstalling the newest version of anaconda and see what that does.

@mmcky

**Comment by mmcky** [comment]
thanks @duncanhobbs please let me know how you go.

**Comment by duncanhobbs** [comment]
I reinstalled the 2019-07 build of anaconda and set up the environment again.

All of the correct versions of packages for work with Python 3.5 and above appear to be installed.

I checked that all the tests pass from the master branch as well.

I always use `conda activate` so I don't think that is the issue.

I think either telling people to use the latest version of anaconda or `create -n quantecon-dev python=3.7 ` will fix this problem.

I think `create -n quantecon-dev python=3.7 ` is easier than installing a new version or upgrading to a new version of anaconda, but it seems like either one works.

@mmcky
